pages/api/skills/[id].js

```diff
@@ -0,0 +1,40 @@
+import { PrismaClient } from '@prisma/client';
+import { authenticated } from '../../../utils';
+
+const prisma = new PrismaClient();
+
+export default authenticated(async function handler(req, res) {
+  let { id } = req.query;
+  id = parseInt(id);
+
+  if (req.method === 'GET') {
+    try {
+      const result = await prisma.skills.findUnique({
+        where: {
+          id,
+        },
+      });
+      res.status(200).json(result);
+    } catch (error) {
+      res.status(500).json({ message: 'Could not find the skill' });
+    }
+  } else if (req.method === 'PATCH') {
+    const updatedSkill = req.body;
+
+    try {
+      const result = await prisma.skills.update({
+        where: {
+          id,
+        },
+        data: updatedSkill,
+      });
+      res.status(200).json(result);
+    } catch (error) {
+      if (error.code === 'P2003') {
+        res.status(500).json({ message: 'hero id is not found' });
```

Comment:
```suggestion
      if (error.code === 'P2003') {
        res.status(500).json({ message: 'hero id is not found' });
```

untuk not found kita bisa pake 404 yak mas @berry1611